dynamic-demo-plugin/oc-manifest.yaml

```diff
@@ -79,6 +79,8 @@ apiVersion: console.openshift.io/v1alpha1
 kind: ConsolePlugin
 metadata:
   name: console-demo-plugin
+  annotations:
+    console.openshift.io/use-i18next: true
```

Comment:
We should update the README for the demo plugin to discuss opting into i18n.

Reply:
done